leftwm/src/bin/leftwm-check.rs

```diff
@@ -41,6 +47,20 @@ async fn main() -> Result<()> {
         "\x1b[0;94m::\x1b[0m LeftWM git hash: {}",
         git_version::git_version!(fallback = option_env!("GIT_HASH").unwrap_or("NONE"))
     );
+    if matches.occurrences_of("migrate") >= 1 {
+        println!("\x1b[0;94m::\x1b[0m Migrating configuration . . .");
+        let path = BaseDirectories::with_prefix("leftwm")?;
+        let ron_file = path.place_config_file("config.ron")?;
+        let toml_file_path = path.place_config_file("config.toml")?;
```

Comment:
This is just a little OCD, can these be named similar (e.g. with _path or not) as they are the same (except the ron and toml)

Reply:
did it the other way round and removed the `let toml_file_str` by moving the function calls to the variable move in the call of `load_from_file`